<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>viewoptions</class>
 <widget class="QDialog" name="viewoptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>169</width>
    <height>96</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>View Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="cbMetaData">
     <property name="text">
      <string>Show MetaData</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbCategorized">
     <property name="text">
      <string>Show Categorized</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbScientific">
     <property name="text">
      <string>Show Scientific</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
